apps/ledger-live-desktop/src/renderer/components/ErrorBody.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { BoxedIcon, Text } from "@ledgerhq/react-ui";
+import { withV3StyleProvider } from "~/renderer/styles/StyleProviderV3";
+import styled from "styled-components";
+
+export const ErrorTitle = styled(Text).attrs({
+  color: "neutral.c100",
+  textAlign: "center",
+})``;
+
+export const ErrorDescription = styled(Text).attrs({
+  color: "neutral.c80",
+  whiteSpace: "pre-wrap",
+  textAlign: "center",
+})``;
+
+/** Renders an error icon, title and description */
+export const ErrorBody: React.FC<{
+  /**
+   * react node to render instead of the icon, at the top
+   * */
+  top?: React.ReactNode | null | undefined;
+  /**
+   * icon to render inside a box
+   */
+  Icon?:
```

Comment:
[ASK]: Why `Icon` is the only props with a uppercase ?

Reply:
The type of `Icon` is a component type, so as a convention we uppercase the first letter